File: C++/0009. Consecutive Integer.cpp
// we have defined the necessary header files here for this problem.
// If additional header files are needed in your program, please import here.

int main()
{  
  // please define the C++ input here. For example: int a,b; cin>>a>>b;;  
  // please finish the function body here.  
  // please define the C++ output here. For example:cout<<____<<endl; 
    int m;
    while(cin>>m){
        while(m>1){
            if(m%2){
                cout<<"YES"<<endl;
                return 0;
            }
            m /= 2;
        }
        cout<<"NO"<<endl;
    }
    return 0;
}

File: Python/0009. Consecutive Integer.py
# If you need to import additional packages or classes, please import here.
import math
def func():
    # please define the python3 input here. 
    # For example: a,b = map(int, input().strip().split())
    # please finish the function body here.
    # please define the python3 output here. For example: print().
    while True:
        try:
            m = int(input())
            while m > 1:
                if m%2:
                    print("YES")
                    return
                else:
                    m //= 2
            print("NO")
            return
            
        except EOFError:
            break
    
if __name__ == "__main__":
    func()
